pallets/bridge/src/lib.rs

```diff
@@ -1,139 +1,190 @@
+				self.status = ProposalStatus::Rejected;
+				ProposalStatus::Rejected
+			} else {
+				ProposalStatus::Initiated
+			}
 		}
-	}
 
-	/// Returns true if the proposal has been rejected or approved, otherwise false.
-	fn is_complete(&self) -> bool {
-		self.status != ProposalStatus::Initiated
-	}
+		/// Returns true if the proposal has been rejected or approved, otherwise false.
+		fn is_complete(&self) -> bool {
+			self.status != ProposalStatus::Initiated
+		}
 
-	/// Returns true if `who` has voted for or against the proposal
-	fn has_voted(&self, who: &A) -> bool {
-		self.votes_for.contains(&who) || self.votes_against.contains(&who)
-	}
+		/// Returns true if `who` has voted for or against the proposal
+		fn has_voted(&self, who: &A) -> bool {
+			self.votes_for.contains(&who) || self.votes_against.contains(&who)
+		}
 
-	/// Return true if the expiry time has been reached
-	fn is_expired(&self, now: B) -> bool {
-		self.expiry <= now
+		/// Return true if the expiry time has been reached
+		fn is_expired(&self, now: B) -> bool {
+			self.expiry <= now
+		}
 	}
-}
 
-impl<AccountId, BlockNumber: Default> Default for ProposalVotes<AccountId, BlockNumber> {
-	fn default() -> Self {
-		Self {
-			votes_for: vec![],
-			votes_against: vec![],
-			status: ProposalStatus::Initiated,
-			expiry: BlockNumber::default(),
+	impl<AccountId, BlockNumber: Default> Default for ProposalVotes<AccountId, BlockNumber> {
+		fn default() -> Self {
+			Self {
+				votes_for: vec![],
+				votes_against: vec![],
+				status: ProposalStatus::Initiated,
+				expiry: BlockNumber::default(),
+			}
 		}
 	}
-}
 
-pub trait Config: system::Config {
-	type Event: From<Event<Self>> + Into<<Self as frame_system::Config>::Event>;
-	/// Origin used to administer the pallet
-	type BridgeCommitteeOrigin: EnsureOrigin<Self::Origin>;
-	/// Proposed dispatchable call
-	type Proposal: Parameter + Dispatchable<Origin = Self::Origin> + EncodeLike + GetDispatchInfo;
-	/// The identifier for this chain.
-	/// This must be unique and must not collide with existing IDs within a set of bridged chains.
-	type BridgeChainId: Get<BridgeChainId>;
-
-	type ProposalLifetime: Get<Self::BlockNumber>;
-}
+	#[pallet::pallet]
+	#[pallet::generate_store(pub(super) trait Store)]
+	pub struct Pallet<T>(_);
+
+	#[pallet::config]
+	pub trait Config: frame_system::Config {
+		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
+		/// Origin used to administer the pallet
+		type BridgeCommitteeOrigin: EnsureOrigin<Self::Origin>;
+		/// Proposed dispatchable call
+		type Proposal: Parameter
+			+ Dispatchable<Origin = Self::Origin>
+			+ EncodeLike
+			+ GetDispatchInfo;
+		/// The identifier for this chain.
+		/// This must be unique and must not collide with existing IDs within a set of bridged chains.
+		#[pallet::constant]
+		type BridgeChainId: Get<BridgeChainId>;
+
+		#[pallet::constant]
+		type ProposalLifetime: Get<Self::BlockNumber>;
+	}
+
+	#[pallet::storage]
```

Comment:
Could you move the storage section down a bit after `Error<T>` so that we can get better diff for tracking? Thanks